website/pages/docs/from-1.5/onlyexcept.mdx

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+page_title: Only Except - HCL Configuration Language
+sidebar_title: Only Except
+description: >-
+  Only and Except can be used as a command line argument to selectively run
+  builds. Only and Except can also be used in a provisioner to not run it for a
+  source.
+---
+
+# Only and Except
+
+`only` and `except` are keywords used to filter what runs in your Packer build,
+they can be seen as a command line argument:
+
+`@include 'commands/except.mdx'`
+
+`@include 'commands/only.mdx'`
+
+They can also be seen in a template:
```

Comment:
```suggestion
They can also be seen in a template to run or skip provisioner and post-processors for a specific source:
```
Do you think this way makes it easier to understand earlier that these ones are to only/except sources instead os builds?

Reply:
Good idea yes ! Adding this